### PR TITLE
Updated test drive page VsCode instructions

### PR DESCRIPTION
**Pull request by ldps21**
_Description of what this PR is changing or adding, and why:_
The test drive page is instructing to install the dart extension which was by
default installed previously by the VsCode Flutter extension.
_Issues fixed by this PR (if any):_
https://github.com/flutter/website/issues/10856
_PRs or commits this PR depends on (if any):_

## Presubmit checklist

- [ ] This PR is marked as draft with an explanation if not meant to land until a future stable release.
- [x] This PR doesn’t contain automatically generated corrections (Grammarly or similar).
- [x] This PR follows the [Google Developer Documentation Style Guidelines](https://developers.google.com/style) — for example, it doesn’t use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [x] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/main/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.

### CODE REVIEW

**Comment by ldps21** [comment]
Screenshot 2024-07-07 at 18 25 34
Here you can see the updated text.

File: src/_includes/docs/install/test-drive/vscode.md [hunk 1]
```diff
@@ -2,9 +2,8 @@
 
 ### Create a sample Flutter app {:#create-app-vs-code}
 
-1. To use the DevTools from VS Code, you need the [Dart extension](https://marketplace.visualstudio.com/items?itemName=Dart-Code.dart-code).
-   To easily debug your application you should also install
-   the [Flutter extension](https://marketplace.visualstudio.com/items?itemName=Dart-Code.flutter).
+1. To use the DevTools from VS Code, install the [Flutter extension](https://marketplace.visualstudio.com/items?itemName=Dart-Code.flutter).
+   This will also automatically install the [Dart extension](https://marketplace.visualstudio.com/items?itemName=Dart-Code.dart-code).With this extensions, you will be able to easily debug you application.
```
Comment (sfshaza2) [inline src/_includes/docs/install/test-drive/vscode.md:6]:
```suggestion
1. To use Flutter DevTools from VS Code, install the
    [Flutter extension](https://marketplace.visualstudio.com/items?itemName=Dart-Code.flutter).
   This also automatically installs the
   [Dart extension](https://marketplace.visualstudio.com/items?itemName=Dart-Code.dart-code).
   With these extensions, you can debug you application.
```

**Review by sfshaza2 — APPROVED** [review]
lgtm with some minor reformatting. Thanks, @ldps21!

**Comment by ldps21** [comment]
@sfshaza2 Should I submit a PR to add myself to the [AUTHORS](https://github.com/flutter/website/blob/02219cd90ffbc20ee5cc730a673ada8eaab89973/AUTHORS#L4) file?

**Pull request merged**